frontend/src/common/analytics/events.ts

```diff
@@ -121,4 +122,26 @@ export enum EVENTS {
   CG_VIEW_CELLGUIDE_PAGE_CLICKED = "CG_VIEW_CELLGUIDE_PAGE_CLICKED",
   CG_OPEN_INTEGRATED_EMBEDDING_CLICKED = "CG_OPEN_INTEGRATED_EMBEDDING_CLICKED",
   SUBMIT_CORRECTION_CLICKED = "SUBMIT_CORRECTION_CLICKED",
+  // DIFFERENTIAL EXPRESSION
```

Comment:
New analytics events

Reply:
Did you generate the comments with AI? What was the prompt?

Reply:
I gave it amanda's analytics ticket xDD